lib/ex_unit/lib/ex_unit/on_exit_handler.ex

```diff
@@ -2,20 +2,42 @@ defmodule ExUnit.OnExitHandler do
   @moduledoc false
 
   @name __MODULE__
-  @supervisor 2
-  @on_exit 3
   @ets_opts [:public, :named_table, read_concurrency: true, write_concurrency: true]
 
+  # ETS column numbers
+  @pid 1
+  @supervisor 3
+  @on_exit 4
+
   use Agent
 
+  @type manager :: ExUnit.EventManager.use(Agent)
```

Comment:
Should this be `ExUnit.EventManager.manager()`?

Reply:
fixed. thank you for all your input